mmv1/products/alloydb/Cluster.yaml

```diff
@@ -156,11 +198,38 @@ properties:
         required: true
         sensitive: true
   - !ruby/object:Api::Type::NestedObject
-    name: 'automatedBackupPolicy'
+    name: 'continuousBackupConfig'
+    default_from_api: true
     description: |
-      The automated backup policy for this cluster.
+      The continuous backup config for this cluster.
+
+      If no policy is provided then the default policy will be used. The default policy takes one backup a day and retains backups for 14 days.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: enabled
+        send_empty_value: true
+        description: |
+          Whether continuous backup recovery is enabled. If not set, defaults to true.
```

Comment:
what would you think about having `default_value: true` on this field if the default is true?

Reply:
This should be fine and likely won't change. Added it.